src/main/java/com/Novi/TechItEasy/models/CI_Module.java

```diff
@@ -0,0 +1,67 @@
+package com.Novi.TechItEasy.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.*;
+
+import java.util.List;
+
+@Entity
+@Table(name = "CI_Modules")
+public class CI_Module {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false, updatable = false)
+    private Long id;
+
+    @Column(name = "name")
```

Comment:
Je geeft hier aan dat je in de database een andere kolom-naam wilt gebruiken dan de variabele-naam die hier al staat, maar je geeft dezelfde naam op. Effectief is deze annotatie dus zinloos nu.